libkbfs/interfaces.go

```diff
@@ -365,7 +371,9 @@ type KBPKI interface {
 	// HasVerifyingKey returns nil if the given user has the given
 	// VerifyingKey, and an error otherwise.
 	HasVerifyingKey(ctx context.Context, uid keybase1.UID,
-		verifyingKey VerifyingKey, atServerTime time.Time) error
+		verifyingKey VerifyingKey, atServerTime time.Time,
+	        checkUnverified bool) error
```

Comment:
If you keep it here, you should explain unverified keys (and their dangerousness) in the comment above.